ganga/GangaCore/Lib/Virtualization/Singularity.py

```diff
@@ -0,0 +1,27 @@
+##########################################################################
+# Ganga Project. http://cern.ch/ganga
+#
+##########################################################################
+from GangaCore.GPIDev.Schema import Schema, Version, SimpleItem
+from GangaCore.GPIDev.Adapters.IVirtualization import IVirtualization
+import GangaCore.Utility.logging
+
+logger = GangaCore.Utility.logging.getLogger()
+
+class Singularity(IVirtualization):
+
+    """
+    Handles all the config related to Singularity
+    """
+    _name = 'Singularity'
+    _schema = Schema(Version(1, 0), {
+        'imageUrl': SimpleItem(defvalue="", doc='Link to the container image')
```

Comment:
This attribute seems ubiquitous across virtualisation clases so should probably exist in the `IVirtualization` base class?

Reply:
moved to IVirtualization class